locust/test/test_main.py

```diff
@@ -191,6 +192,49 @@ def my_task(self):
             self.assertIn("Shutting down (exit code 0)", stderr)
             self.assertEqual(0, proc.returncode)
 
+    def test_percentile_parameter(self):
+        with temporary_file(
+            content=textwrap.dedent(
+                """
+            from locust import User, task, constant, events
+            from locust.stats import PERCENTILES_TO_CHART
+            PERCENTILES_TO_CHART[0] = 0.9
+            PERCENTILES_TO_CHART[1] = 0.4
+            class TestUser(User):
+                wait_time = constant(3)
+                @task
+                def my_task(self):
+                    print("running my_task()")
+        """
+            )
+        ) as file_path:
+            proc = subprocess.Popen(["locust", "-f", file_path], stdout=PIPE, stderr=PIPE, text=True)
```

Comment:
Add something to the locustfile that actually makes requests, add `--autostart` to make actually run, and check the exit code or something.

Reply:
I'll update test_main